platform/kube/controller.go

```diff
@@ -579,10 +579,10 @@ func (c *Controller) HostInstances(addrs map[string]bool) []*model.ServiceInstan
 }
 
 const (
-	// IstioServiceAccountPrefix is always the prefix for Istio service accounts.
-	IstioServiceAccountPrefix = "istio:"
-	// LocalDomain is the domain name Istio service account uses for internal traffic.
-	LocalDomain = "cluster.local"
+	// the URI scheme used to encode a Kubernetes service account
+	uriScheme = "spiffe"
+	// the domain name Istio service account uses for internal traffic
+	localDomain = "cluster.local"
```

Comment:
why are we not following kubernetes standard: `<name>.<namespace>.svc.<dns-domain>`?